templates/monorepo/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "./tsconfig.build.json",
+  "include": ["packages", "types", "scripts", "example"],
+  "compilerOptions": {
+    "allowJs": false,
+    "baseUrl": ".",
+    "typeRoots": ["./node_modules/@types", "./types"],
+    "paths": {
+      "@mono/utils": ["packages/eval/src"],
+      "@mono/react": ["packages/react/src"],
+      "$test/*": ["test/*"]
```

Comment:
I copied this file and the paths setup from reach/ui. They have some global test helpers inside of a top level test folder. They forked tsdx and simplified it for their use case. IIRC they tweaked jest command to look at the entire project instead of once per module. I didn’t get that far, I basically got this working and pushed it up immediately. So it makes sense that there are extraneous configs